components/Header.vue

```diff
@@ -115,154 +198,263 @@
-                <div class="relative flex items-center ring-1 ring-slate-900/10 rounded-lg shadow-sm p-2 text-slate-700 font-semibold dark:bg-slate-600 dark:ring-0 dark:highlight-white/5 dark:text-slate-200">
-                  <SunIcon :class="selected =='Light' ?  'h-6 w-6' : 'hidden'" /> 
-                  <MoonIcon :class="selected == 'Dark' ?  'h-6 w-6' : 'hidden'" /> 
-                  <DesktopComputerIcon :class="selected == 'System' ?  'h-6 w-6' : 'hidden'" />
-                  <div class="ml-2">{{selected}}</div>
-                  <svg class="w-5 h-6  text-slate-400" fill="none"><path d="m15 11-3 3-3-3" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"></path>
+                <label
+                  class="text-base font-medium text-gray-900 dark:text-MITRE-silver"
+                  >Switch Theme</label
+                >
+                <div
+                  class="dark:highlight-white/5 relative flex items-center rounded-lg p-2 font-semibold text-slate-700 shadow-sm ring-1 ring-slate-900/10 dark:bg-slate-600 dark:text-slate-200 dark:ring-0"
+                >
+                  <SunIcon
+                    :class="selected == 'Light' ? 'h-6 w-6' : 'hidden'"
+                  />
+                  <MoonIcon
+                    :class="selected == 'Dark' ? 'h-6 w-6' : 'hidden'"
+                  />
+                  <DesktopComputerIcon
+                    :class="selected == 'System' ? 'h-6 w-6' : 'hidden'"
+                  />
+                  <div class="ml-2">
+                    {{ selected }}
+                  </div>
+                  <svg class="h-6 w-5 text-slate-400" fill="none">
+                    <path
+                      d="m15 11-3 3-3-3"
+                      stroke="currentColor"
+                      stroke-width="2"
+                      stroke-linecap="round"
+                      stroke-linejoin="round"
+                    />
                   </svg>
-                  <select id="theme" class="absolute appearance-none inset-0 w-full h-full opacity-0 dark:bg-slate-600 dark:text-white" @change="switchSelect($event)">
-                  <option :selected="selected == 'Light'">Light</option>
-                  <option :selected="selected == 'Dark'">Dark</option>
-                  <option :selected="selected == 'System'">System</option>
+                  <select
+                    id="theme"
+                    class="absolute inset-0 h-full w-full appearance-none opacity-0 dark:bg-slate-600 dark:text-white"
+                    @change="switchSelect($event)"
+                  >
+                    <option :selected="selected == 'Light'">Light</option>
+                    <option :selected="selected == 'Dark'">Dark</option>
+                    <option :selected="selected == 'System'">System</option>
                   </select>
                 </div>
               </div>
```

Comment:
We should look into the ease of adding this into a separate component to make code cleaner

Reply:
After further discussion we will come back to this at a later date.